chucks-pick3-web/angular-app/src/app/app.module.ts

```diff
@@ -22,11 +22,14 @@ import {InfoGenerateComponent} from 'src/app/components/info-generate.component/
 import {InfoRequirementsComponent} from 'src/app/components/info-requirements.component/info-requirements.component';
 import {ClickOutsideModule} from 'ng-click-outside';
 import {ScrollEventModule} from 'ngx-scroll-event';
-import {SubNavBarService} from 'src/app/services/show-sub-nav-bar.service/sub-nav-bar.service';
+import {SubNavBarService} from './services/show-sub-nav-bar.service/sub-nav-bar.service';
```

Comment:
Interesting that you are just now adding the SubNavBarService. How was this able to work before?

Reply:
I'm not sure how this wasn't there before, Everytime I would test it the components were there. I think this might have happened after I completed the merge conflicts on it?